content/scripts/build.js

```diff
@@ -715,7 +722,7 @@ class Builder {
     // First regroup ALL URLs into buckets per locale.
     const byLocale = {};
     const mostModified = {};
-    for (let [uri, data] of Object.entries(this.allTitles)) {
+    for (const [uri, data] of this.allTitles) {
```

Comment:
I switched from `let` to `const` simply because of consistency. If you ask me, I generally prefer `let` because it's shorter and you have to use it anyway if you ever need to change its value (beyond a mutation).